core/src/test/java/com/github/dozermapper/core/functional_tests/EnumMapping_WithExceptionsLoggedTest.java

```diff
@@ -60,8 +60,8 @@
     public void testEnumMapsToNonexistEnumValue() {
         LOG.error("WithExceptionsLoggedTest; 'IllegalArgumentException: No enum constant com.github.dozermapper.core.vo.enumtest.DestType.BAR'");
 
-        testEnumMapsToNonexistEnumValueEE.expectMessage("No enum constant com.github.dozermapper.core.vo.enumtest.DestType.BAR");
-        testEnumMapsToNonexistEnumValueEE.expect(IllegalArgumentException.class);
+        testEnumMapsToNonexistEnumValueEE.expectMessage("Cannot convert [BAR] to enum of type class com.github.dozermapper.core.vo.enumtest.DestType");
+        testEnumMapsToNonexistEnumValueEE.expect(MappingException.class);

```

Comment:
Whys this changed?

Reply:
IllegalArgumentException is caught in com.github.dozermapper.core.converters.EnumConverter#convert and then it is wrapped in MappingException and threw out.
So the expected exception should be MappingException, not the original IllegalArgumentException.